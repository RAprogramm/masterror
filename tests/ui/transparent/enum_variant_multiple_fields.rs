// SPDX-FileCopyrightText: 2025 RAprogramm <andrey.rozanov.vl@gmail.com>
//
// SPDX-License-Identifier: MIT

use masterror::Error;

#[derive(Debug, Error)]
enum TransparentEnumFail {
    #[error(transparent)]
    Variant(String, String)
}
