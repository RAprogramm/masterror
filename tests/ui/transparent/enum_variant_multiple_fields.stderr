error: #[error(transparent)] requires exactly one field
  --> tests/ui/transparent/enum_variant_multiple_fields.rs:9:5
   |
 9 | /     #[error(transparent)]
10 | |     Variant(String, String)
   | |___________________________^

error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/ui/transparent/enum_variant_multiple_fields.rs:11:2
   |
11 | }
   |  ^ consider adding a `main` function to `$DIR/tests/ui/transparent/enum_variant_multiple_fields.rs`
